aws-datastore/src/test/java/com/amplifyframework/datastore/appsync/AppSyncConflictUnhandledErrorTest.java

```diff
@@ -119,9 +119,8 @@ public void conflictErrorExtractedIfPresent() {
             AppSyncConflictUnhandledError.findFirst(Note.class, response.getErrors());
         assertNotNull(conflictUnhandledError);
 
-        // TODO: The JSON document has '1601499066604' as the time. These differ by 26604,
-        // also the TimeUnit appears to be wrong. Should be MILLISECONDS.
-        Temporal.Timestamp lastChangedAt = new Temporal.Timestamp(1601499040000L, TimeUnit.SECONDS);
+        // TODO: the TimeUnit appears to be wrong. Should be MILLISECONDS?
```

Comment:
Accidentally fixed one of the bugs here. Appears to have been caused by an overflow.

The milliseconds issue is unrelated. That's about AppSync using `AWSTimestamp` as:
> the number of seconds that have elapsed since 1970-01-01T00:00Z

But then it returns a value in milliseconds in the conflict handler data, in production 🤷 .